docs/farming-&-staking/farming/additional-guides/port-forwarding.md

```diff
@@ -17,7 +17,7 @@ If you are using Linux and enabled `ufw` (firewall), make sure you have opened u
 
 First before forwarding ports it is important to understand what that actually means. We would highly suggest reading some information on the topic, here is a guide we find helpful https://www.geeksforgeeks.org/port-forwarding-on-router-and-why-do-we-need-it/
 
-Forwarding ports on your router will open up one specific lane on your router to allow external traffic to flow in or out. This is usually automatically done for most applications such as port 80 for File Transfer Protocol, in our case we will need to do this manually.
+Forwarding ports on your router will open up one specific lane on your router to allow external traffic to flow in or out. This is usually automatically done for most applications such as port 80 for Hypertext Transfer Protocol, in our case we will need to do this manually.
```

Comment:
```suggestion
Forwarding ports on your router will open up one specific lane on your router to allow external traffic to flow in or out. This is usually automatically done for most applications such as port 80 for File Transfer Protocol, in our case we will need to do this manually.
```

I believe in this case we need to open port for FTP rather than HTTP.

Reply:
It's just an example in this context, so the port number along with the protocol doesn't matter at all. But I've changed it since the port 80 is the standard one for HTTP. If you prefer FTP, I would suggest the standard FTP port 21, just to be consistent. :)

Reply:
> I believe in this case we need to open port for FTP rather than HTTP.

It's just an example in this context, so the port number along with the protocol doesn't matter at all. But I've changed it since the port 80 is the standard one for HTTP. If you prefer FTP, I would suggest the standard FTP port 21, just to be consistent. :)